crates/rpc/src/pathfinder/unstable.rs

```diff
@@ -0,0 +1,11 @@
+//! This RPC API should **always** be considered unstable and for internal use
+//! only. It's main purpose is to aid integration testing so expect it to change
+//! anythime!
```

Comment:
Typo... I know it's irrelevant, but happened to see it :)

Reply:
Thanks, I haven't noticed both of them for some reason :facepalm: